setup.py

```diff
@@ -124,10 +126,9 @@ def run(self):
               'sphinx-autobuild',
               'sphinx-rtd-theme',
               'sphinx_bootstrap_theme~=0.4.0',
-              'pydocstyle~=1.1.1',
-              'pylama~=7.3.3',
-              'pylama_pylint~=3.0.1',
-              'radon~=1.5.0'
+              'pydocstyle',
```

Comment:
@macartur we do not need `pydocstyle` (neither `radon`) since they are already required by yala.